modules/standard/Regex.chpl

```diff
@@ -963,12 +1039,14 @@ record regex {
     return str;
   }
 
+  // TODO this could use chpl__serialize to get the pattern and options
```

Comment:
Since I don't think we want to call `chpl__serialize` directly, maybe update this comment to refer to `_serialize`?